### PR TITLE
Tempe bwa index updates

### PR DESCRIPTION
**Pull request by bryce-turner**

### CODE REVIEW

File: tempe/README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ while the reference used for RNA does not have ALT-contigs as STAR is not ALT-aw
 ## Script Usage Order
 
 * sbatch tempe/create_genome_reference.sh tempe/tempe_resources.ini
-  * bash shared_resource_creation_scripts/create_bwa_genome_index.sh tempe/tempe_resources.ini
+  * bash tempe/create_bwa_genome_index.sh tempe/tempe_resources.ini
```
Comment (PedalheadPHX) [inline tempe/README.md:14]: I'd add a comment
# Uses custom creation script to ensure the genome specific alt file is downloaded so bwa runs in alt-aware mode

File: tempe/create_bwa_genome_index.sh [hunk 2]
```diff
@@ -80,18 +80,18 @@ echo >> README
...
 
+# Determine the expected FASTA sequence filename based on the download link
+FASTA=$REFERENCE_DNA_GENOME_NAME
+
+# Create a symbolic link to the reference genome
+ln -s ../../genome_reference/${FASTA} ${FASTA}
+
 # Copy in the .alt file from bwa.kit
 echo "Copy in the .alt file from bwa.kit" >> README
-cp ../../genome_reference/downloads/bwa.kit/resource-GRCh38/hs38DH.fa.alt GRCh38tgen_decoy_alts_hla.fa.alt
```
Comment (PedalheadPHX) [inline tempe/create_bwa_genome_index.sh:85]: Is this how we did it before, so the user needs to have independently pulled bwakit?

**Review by PedalheadPHX — COMMENTED** [review]
@bryce-turner need to address the two comments, both, since we rely on the relative path to pull the alt file from bwakit, is there some independent script that must be run in advance to make sure bwakit reference is available, needs to be noted in the tempe and phoenix build versions

File: tempe/create_bwa_genome_index.sh (hunk 2 above)
Comment (bryce-turner) [reply tempe/create_bwa_genome_index.sh:85]: We pull bwa.kit during the create_genome_reference.sh script that runs before this, though maybe we want to change this to be a path defined in the ini? e.g. PATH_TO_BWAKIT="/some/path" and we navigate the known structure and or we use/ask for the full path to the alt.

Future implementation might be to create the alt ourselves, though the curated file in bwa.kit is likely more "correct" than an automated approach.

File: tempe/README.md (hunk 1 above)
Comment (bryce-turner) [reply tempe/README.md]: Updated in [1b8630f](https://github.com/tgen/jetstream_resources/pull/72/commits/1b8630f8e9d810816400fa62860ac052829321ec)

**Comment by bryce-turner** [comment]
Merged after internal discussion

**Pull request merged**